bower.json

```diff
@@ -7,12 +7,13 @@
   "copyright": "hmgibson23",
   "main": "scribe-plugin-drag-and-drop.js",
   "dependencies": {
-    "scribe": "~1.3.2",
-    "scribe-common": "~0.0.11",
-    "lodash-amd": "~2.4.1",
     "requirejs": "~2.1.18"
   },
-  "devDependencies": {},
+  "devDependencies": {
```

Comment:
They are needed for the test page (which isn't being used currently but I didn't want to break it in a non-obvious way).